python/ray/data/_internal/compute.py

```diff
@@ -315,6 +315,14 @@ def get_compute(compute_spec: Union[str, ComputeStrategy]) -> ComputeStrategy:
         raise ValueError("compute must be one of [`tasks`, `actors`, ComputeStrategy]")
 
 
+def is_task_compute(compute_spec: Union[str, ComputeStrategy]) -> bool:
+    return (
+        not compute_spec
+        or compute_spec == "tasks"
+        or isinstance(compute_spec, TaskPoolStrategy)
+    )
```

Comment:
Nice, I was thinking about adding a similar helper for `is_actor_compute()` in the UDF args PR.